web/wsgi.py

```diff
@@ -52,9 +52,9 @@ def runwsgi(func):
             return runscgi(func)
     
     
-    server_addr = validip(listget(sys.argv, 1, ''))
-    if 'PORT' in os.environ: # e.g. Heroku
```

Comment:
Why was comment changed. How does this change affect existing users? Thinking Heroku since it is mentioned there?